Dockerfile

```diff
@@ -12,24 +12,17 @@ WORKDIR /app/
 COPY requirements.txt requirements.txt
 RUN pip install --upgrade pip && pip install setuptools -U && pip install --user -r requirements.txt
 
-# Install a custom OPAL, if requested
-COPY custom /custom
-
-RUN if [ -f /custom/custom_opal.tar.gz ]; \
-	then \
-		cd /custom && \
-		tar xzf custom_opal.tar.gz && \
-		pip install --user packages/opal-common packages/opal-client && \
-		cd / && \
-		rm -rf /custom ; \
-	fi
-
 COPY horizon setup.py MANIFEST.in ./
 RUN python setup.py install --user
 
 FROM golang:bullseye as OPABuildStage
 
-COPY custom /custom
+# Check if the source directory exists before copying
+RUN if [ -d ./permit-opa/custom ]; then \
```

Comment:
This is not in keeping with the `Makefile` - which expects `CUSTOM_OPA` envar, packs it into .tar.gz, and then builds the docker.
It's better if the CI builds the sidecar same way we build it locally (or at least be similar as possible)

Reply:
I'm not using at all Makfile at the CI.
The Makefile would be for future ticket which I going to make this Makefile for local build of PDP.

Reply:
Not sure I get what you're saying.
The Makefile already builds local PDP - I (and probably others) use it when I develop to test changes locally.
I don't think your CI must use the Makefile (although it might have been nice) - but I think it's messy that:
1. There are 2 different ways to set the custom OPA the Dockerfile uses (either having .tar.gz or putting it in a directory)
2. Makefile finds the custom OPA in `$CUSTOM_OPA` then packs it into Dockerfile's expected .tar.gz, while CI workflow checks it out to a directory.

If you mean that you agree that's not ideal, but prefer to work on it as a separate ticket - I guess that's legit :)

Reply:
The CI will use his way, the makefile still be fine for local build.
At the other future ticket, I'll make this makefile 1-click.